source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -602,6 +602,9 @@ established. In addition, the Pool MUST NOT prevent other threads from checking
 out `Connections <#connection>`_ while establishing a `Connection
 <#connection>`_.
 
+If connection establishment fails due to a network error drivers SHOULD mark the
```

Comment:
I can roll back the change in here, I was thinking that it wasn't sufficient to mention the network error in the retryable spec since it would be created at the CMAP layer, but I think I'm influenced by how node does things here.

Reply:
Done, removed the cmap changes!